tests/v2/validation/charts/resources/cis-profiles/cluster-scan-template.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cis.cattle.io/v1
+kind: ClusterScan
+metadata:
+  name: %s #string
+  annotations:
+    {}
+#    key: string
+spec:
```

Comment:
isn't there a way to get the values.yaml from the chart instead of hard coding this? Its definitely in rancher somewhere, because the data is not manually entered by a user. I worry that hard coding it here will result in us missing updates if rancher adds any later.